htsp/src/main/java/ie/macinnes/htsp/messages/BaseEventResponse.java

```diff
@@ -204,6 +215,11 @@ public ContentValues toContentValues(long channelId) {
             // If we have only description, use it.
             values.put(TvContract.Programs.COLUMN_SHORT_DESCRIPTION, mDescription);
         }
+        
+        if (mAgeRating != INVALID_INT_VALUE) {
+            TvContentRating rating = TvContentRating.createRating("com.android.tv", "DVB", "DVB_" + mAgeRating);
```

Comment:
I haven't looked, as none of my EPG sources include age details, will these always line up? E.g. will createRating() throw or return null? E.g. what happens if we receive an ageRating of 3, or 19 (which isn't in the list of DVB_* ratings Android understands). It's probably unlikely, and I can't think of an example, but different countries doing things I haven't expected has bit me before :D